.github/workflows/builds.yml

```diff
@@ -352,7 +349,8 @@ jobs:
           -DCMAKE_PREFIX_PATH="${{ env.DEPENDENCY_DIR }}"
           -DPython_EXECUTABLE=${{ env.DEPENDENCY_DIR }}/bin/python3
           -DCMAKE_INSTALL_PREFIX="${{ env.INSTALL_DIR }}"
-          -DACTS_BUILD_EXAMPLES_PYTHON_BINDINGS=OFF
+          -DACTS_BUILD_PLUGIN_GEOMODEL=ON
+          -DACTS_BUILD_EXAMPLES_PYTHON_BINDINGS=ON
```

Comment:
I don't know why this was off, but I think I'd prefer if we continue testing this build on macOS as well.

Reply:
I think this should be enabled with the preset?

Reply:
I don't believe it is. The Linux jobs also enable this manually I believe.

Reply:
for GitHub Actions it should be `ON` by default https://github.com/acts-project/acts/blob/1d0a641dfa0a1f4ff3fb898df7c3b3d1642c18d3/CMakePresets.json#L19C18-L19C53

I guess it was switched `OFF` because we didn't built that before https://github.com/acts-project/acts/pull/3135 either